.github/workflows/create-release.yml

```diff
@@ -16,7 +15,7 @@ jobs:
       - uses: './createReleaseAction'
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          head: feat/release-pr
+          head: develop
           base: master
-          title: Testing create PR
-          reviewers: '@nugit/front-devs,team-engineering-x'
+          title: Next Release
+          reviewers: 'moroine,limtingzhi,tuannugit'
```

Comment:
Are we unable to use teams, or is it this repo that's why it can't work?

Reply:
No we're unable to use teams 😢 looks like we don't have permission to see the team members... Maybe because this repo is public 🤔

Reply:
Yeah maybe, we can try on private repos later on.